Typstify/Typstify/ContentView.swift

```diff
@@ -17,7 +17,7 @@ import ProjectNavigator
 // MARK: -
 // MARK: UUID serialisation
 
-extension UUID: @retroactive RawRepresentable {
```

Comment:
Is this a custom attribute?
I thought it was not necessary and deleted it.

Reply:
Yeah, it's not needed.

Reply:
There's a warning

> Extension declares a conformance of imported type 'UUID' to imported protocol 'RawRepresentable'; this will not behave correctly if the owners of 'Foundation' introduce this conformance in the future

if `@retroactive` is removed. Let's just keep it at the moment.